library/model/src/test/kotlin/org/cru/godtools/model/LanguageTest.kt

```diff
@@ -63,5 +64,13 @@ class LanguageTest {
             }
         }
     }
+
+    @Test
+    fun `getForceDisplayName()`() {
```

Comment:
you are testing the `getDisplayName()` function here, not a non-existant `getForceDisplayName()`.

you probably want to name this something like: `getDisplayName() - isForcedName=true`